array_api_tests/meta_tests/test_hypothesis_helpers.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from hypothesis import given
+
+from .. import _array_module as xp
+from .._array_module import _UndefinedStub
+from ..array_helpers import dtype_objects
+from ..hypothesis_helpers import (mutually_promotable_dtype_pairs,
+                                  promotable_dtypes)
+
+UNDEFINED_DTYPES = any(isinstance(d, _UndefinedStub) for d in dtype_objects)
+pytestmark = [pytest.mark.skipif(UNDEFINED_DTYPES, reason="undefined dtypes")]
+
+
+def test_promotable_dtypes():
```

Comment:
I've never thought about how to test strategies before. It's probably a good idea for some of the more complex strategies. Is this how strategies are typically tested in hypothesis itself?

Reply:
Yep, although I'll clean it up now as it could just be top-level `@given()` and `assert in ...` in the method body, instead of whatever set stuff I'm doing.